src/daos/maria/SessionMariaDBDao.ts

```diff
@@ -0,0 +1,15 @@
+import { ISessionDao } from "../ISessionDao";
+import {SessionDto} from "../../model/UserRegistration"
+export class SessionMariaDBDao implements ISessionDao{
+    constructor(){}
```

Comment:
Just one note, I think you need to install the prettier stuff so that all of this will format on a commit